mllib/src/main/scala/org/apache/spark/mllib/classification/SVM.scala

```diff
@@ -33,9 +33,10 @@ import org.apache.spark.rdd.RDD
  * @param weights Weights computed for every feature.
  * @param intercept Intercept computed for this model.
  */
-class SVMModel (
-    override val weights: Vector,
-    override val intercept: Double)
+@Since("0.8.0")
+class SVMModel @Since("1.1.0") (
+    @Since("1.0.0") override val weights: Vector,
+    @Since("0.8.0") override val intercept: Double)
```

Comment:
This means the class is since 0.8, the constructor is since 1.1, `weights` is since 1.0, and `intercept` is since 0.8.